app/pit/build.sbt

```diff
@@ -38,7 +38,7 @@ def common(pv: Version) = AppConfig(
     "org.osgi.framework.storage.clean"        -> "onFirstInit",
     "org.osgi.framework.startlevel.beginning" -> "100",
     "org.osgi.framework.bootdelegation"       -> "*",
-    "edu.gemini.pit.test"                     -> "true",
+    "edu.gemini.pit.test"                     -> "false",
```

Comment:
I don’t recall all the detail, but it seems this should be removed for non test, we check the presence not the value

Reply:
Actually, now that you mention it, I think you may be right.

We should fix that as it's kind of ridiculous.